src/views/CheckoutSuccess.vue

```diff
@@ -1,27 +1,30 @@
 <template>
     <div class="success center">
         <div class="icon-checkmark-circle"></div>
-        <h1>Your {{ requestKindString }}<br>was successfull!</h1>
+        <h1>Your payment<br>was successfull!</h1>
         <div style="flex-grow: 1;"></div>
-        <button @click="close">Back to store</button>
+        <button @click="close" :disabled="!isTxSent">Back to store</button>
```

Comment:
Disable button until `TransactionSender` fires the `transaction-sent` event. This is only 1-2 seconds currently as it's loading the Nimiq library, but will be longer when this component needs to relay the tx to an external server or establish consensus in the future.